homeassistant/components/vicare/climate.py

```diff
@@ -311,8 +311,8 @@ def set_preset_mode(self, preset_mode: str) -> None:
             )
 
         _LOGGER.debug("Current preset %s", self._current_program)
-        if self._current_program and self._current_program != VICARE_PROGRAM_NORMAL:
-            # We can't deactivate "normal"
+        if self._current_program and self._current_program != VICARE_PROGRAM_NORMAL and self._current_program != VICARE_PROGRAM_STANDBY:
+            # We can't deactivate "normal" or "standby"
```

Comment:
use `not in` as also suggested in https://github.com/home-assistant/core/pull/105642/files

Reply:
```suggestion
        if self._current_program and self._current_program not in [
            VICARE_PROGRAM_NORMAL,
            VICARE_PROGRAM_STANDBY,
        ]:
            # We can't deactivate "normal" or "standby"
```

Reply:
Done!